drivers/can/can_sam0.c

```diff
@@ -109,6 +113,12 @@ static int can_sam0_get_core_clock(const struct device *dev, uint32_t *rate)
 
 static void can_sam0_clock_enable(const struct can_sam0_config *cfg)
 {
+	*cfg->mclk |= cfg->mclk_mask;
+
+	GCLK->PCHCTRL[cfg->gclk_id].reg = GCLK_PCHCTRL_CHEN
+					| GCLK_PCHCTRL_GEN(cfg->gclk_gen)
+					;
```

Comment:
Formatting looks off here?

Reply:
How is this a reason for accepting poorly formatted code? What am I missing?

Reply:
I think this is not about poorly formatted code is about style. Since I need to apply the same kind of changes for the whole platform I use the same style for all. The styling always were a problem and always will be since everyone has an opinion about it and they are contradictory most of time. I can accept to change and use your preferences on the CAN driver since you maintain this drivers pretty well. I'll send a fix for this driver.